### PR TITLE
feat(configV4): migrate config file to v4

### PR DESCRIPTION
**Pull request by paologallinaharbur**

### CODE REVIEW

File: oracledb-config.yml.sample [hunk 1]
```diff
@@ -1,40 +1,43 @@
...
+    TABLESPACES: '["tablespace1", "tablespace2"]'
+
+    # Collect extended metrics. If omitted, defaults to false.
+    EXTENDED_METRICS: true
+    # Disable connection pool. Might fix issues with the applciation not being able to execute some queries
+    DISABLE_CONNECTION_POOL: false
+
+    # A custom metrics query will run the custom query, then save the columns as
+    # metrics on the OracleCustomSample event type.
+    CUSTOM_METRICS_QUERY: >-
```
Comment (carlossscastro) [inline oracledb-config.yml.sample:34]: Unrelated to v4 conversion but just noticed our default sample file has this custom query uncommented by default.
We should comment this out not to ship with an active custom query.

**Review by carlossscastro — CHANGES_REQUESTED** [review]

File: oracledb-config.yml.sample (hunk 1 above)
Comment (paologallinaharbur) [reply oracledb-config.yml.sample]: makes sense, I have commented it out

**Review by carlossscastro — APPROVED** [review]
Perfect!

File: oracledb-config.yml.sample [hunk 2]
```diff
@@ -1,40 +1,43 @@
...
+    # True if the monitoring user is a SysOper. If omitted, defaults to false.
+    IS_SYS_OPER: false
+    # A JSON array of tablespaces to collect. If omitted, collects all tablespaces.
+    TABLESPACES: '["tablespace1", "tablespace2"]'
+
+    # Collect extended metrics. If omitted, defaults to false.
+    EXTENDED_METRICS: true
+    # Disable connection pool. Might fix issues with the applciation not being able to execute some queries
+    DISABLE_CONNECTION_POOL: false
+
```
Comment (carlossscastro) [inline oracledb-config.yml.sample:31]: Maybe adding here `MAX_OPEN_CONNECTIONS` which is missing
https://github.com/newrelic/nri-oracledb/blob/master/src/oracledb.go#L31

```
# Maximum number of connections opened by the integration
# MAX_OPEN_CONNECTIONS: 5
```

Comment (alvarocabanas) [reply oracledb-config.yml.sample:31]: Ok sure I will add the flag

Comment (alvarocabanas) [reply oracledb-config.yml.sample:31]: DONE

**Review by alvarocabanas — APPROVED** [review]

**Pull request merged**